faq/using.rst

```diff
@@ -44,3 +44,10 @@ dependecy graph in case OpenSSL is changed we could type:
     [OpenSSL/1.0.2m@conan/stable], [Poco/1.8.1@pocoproject/stable]
 
 So, if OpenSSL is changed, we would need to rebuild it (of course) and rebuild Poco.
+
+A recipe upload from a different machine makes packages outdated without itself changing
```

Comment:
Needs some rewording, the title reads a bit complicated. At least "without changing itself"